packages/mwp-logger-plugin/src/index.js

```diff
@@ -1,19 +1,56 @@
 import logger from './logger';
 
-const onRequestError = (request, err) => {
-	logger.error(
-		{
-			err,
+export function logResponse(request) {
+	const { response, id, server: { app: { logger } } } = request;
+
+	if (response.isBoom) {
+		// response is an Error object
+		logger.error({
+			err: response,
 			...request.raw,
-		},
-		`500 Internal server error: ${err.message}`
-	);
+		});
+	}
+
+	const log = ((response.statusCode >= 500 && logger.error) ||
+		(response.statusCode >= 400 && logger.warn) ||
```

Comment:
should 4xx errors be treated as 'errors' in the logs, or just warning? Actual thrown errors will result in 5xx responses.

Reply:
404s strike me as a special class of errors. in small numbers we really don't care. if a particular 404 is served in high quantity then something is likely wrong.

i guess i'm concerned that logging them as 'warm' might hide problems.

Reply:
Yeah, I think warnings are better. This is also a thing that we can instrument in stackdriver to provide visibility.